README.md

```diff
@@ -27,7 +27,7 @@ Package distribution: [pypi.org/project/dxfeed](https://pypi.org/project/dxfeed/
 **Requirements:** python >= 3.6, pandas
 
 ```python
-pip3 install pandas
+pip3 install pandas, toml
```

Comment:
Why are we insisting user to install this packages separately? We do have them as dependencies in our pyproject.toml, so during regular installation process of our package pip should install them too. 🤔